src/components/Education/SectionList/index.scss

```
@@ -0,0 +1,23 @@
+@import '../../../styles/index.scss';
+
+.section-list__info {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.section-list__university {
+  color: $color-gray-900;
+  font-weight: bold;
+  line-height: 1.2;
+}
+
+.section-list__studyType {
+  color: $color-gray-700;
+}
```

You can use scss power here:
```suggestion

.section-list {
  &__info {
    display: flex;
    align-items: center;
    justify-content: space-between;
  }
  &__university {
    color: $color-gray-900;
    font-weight: bold;
    line-height: 1.2;
  }
  &__studyType {
    color: $color-gray-700;
  }
}
```